test/features/lb/public_lb.feature

```diff
@@ -189,14 +190,14 @@ Feature: Test different tfvars configurations for the lb module with a public ti
         Then terraform resource 'aws_lb' 'default' has changed attributes
             | attr                       | value             |
             #----------------------------|-------------------|
-            | enable_deletion_protection | true              |
-            | enable_http2               | false             |
-            | idle_timeout               | 100               |
+            | enable_deletion_protection | ${bool:True}              |
+            | enable_http2               | ${bool:False}             |
+            | idle_timeout               | ${int:100}               |
```

Comment:
What is the deal with this syntax?

Reply:
This is the interpolation system at work. the `:` character separates the function name from its input value. Without the interpolations, they values would be strings and fail when comparing against boolean and integer values provided by the `terraform plan` return dictionary.

Reply:
I don't mind having the option to be explicit but it would be nice if `true`, `false`, or a number without quotes were treated as a Terraform `bool` or `number` type automatically. Is that possible? Note that terraform has three primitive [types](https://www.terraform.io/docs/configuration/types.html): `string`, `number`, and `bool`. It does not have an `int` type. The `number` type includes both integers and fractional values.

Reply:
Any follow thoughts about this? I think we should discuss this in person. I would like to better understand how the typing works.

Reply:
We can discuss in person. I'm reticent to introduce automatic typing for a few reasons.

1. Doing so for all values on all steps would introduce breaking changes to non-terraform tests (scrum-team-d, iamu, etc.).
2. Doing so for all values on JUST terraform steps would make the behavior inconsistent between modules, and so using them together would be confusing e.g. deploying a web app and running through it with selenium would involve using steps that would behave differently.
3. Previous solutions to this problem involved "string-ifying" json output in an attempt to have consistent-looking tests, but I'm no longer convinced that this is the best approach. It feels more philosophically consistent to test against unmodified data, but I recognize this is a personal preference.

Reply:
One thing that I agree should be added would be `float` support for typing, since you've mentioned that there are possible fractional values that will be produced by terraform.
e.g. `${float: 1.01}`
Java-oriented users might appreciate `${double: 1.01}`
Non-technical people might appreciate `${decimal: 1.01}`

There are other (ezpz) ways that we could expand the interpolation language to accommodate different types of users, but this subject would need to be discussed more thoroughly outside of this comment thread.

Reply:
Another thing that I realize wasn't clear: In the previous output for terraform 0.11 using the `parse-terraform-plan` node package, the `${bool:true}` values would instead have been `computed`. The output for the pyhcl2 package and terraform 0.12 re-organize this information in the output to `true` boolean values, q.e.d. the value conversion.

Reply:
Would it be possible to optionally string-ifying? In reality for AWS and Terraform everything is basically a string anyway.